apps/silverback-drupal/config/sync/core.extension.yml

```diff
@@ -5,8 +5,8 @@ module:
   block: 0
   breakpoint: 0
   ckeditor: 0
-  content_sync: 0
   dblog: 0
+  default_content: 0
```

Comment:
Could you state the reasons why you switched from `content_sync` to `default_content`? For future reference and not repeating ourselves 😄

Reply:
`content_sync`:
- no D9 support yet, I had to patch it hard (btw, I just recall to remove the patch 🙂 https://github.com/AmazeeLabs/silverback-mono/pull/610/commits/434176160c2f4949c6538dc4ecc78fb385e2997f)
- not really maintained
- when you run it, there can be errors and warnings in the output, yet it still somehow woks
- you have to run a command to import the content
- there was an issue with importing content after drupal install cache update

`default_content`:
- was recommended to me by @colorfield, he had good experience with it
- actively maintained by Berdir
- works more stable
- you don't need to run additional commands, content is added when you install a module
- no issues with drupal install cache update

I think it can be added to our recommended modules list.